site/source/user-manual/reset-password.rst

```diff
@@ -17,25 +17,31 @@ Reset Your Password
 
 #. Select "Recover".
 
-   .. figure:: /_static/images/setup/screen0-startfresh_or_recover.jpg
+   .. figure:: /_static/images/setup/screen0-startfresh_or_recover.png
       :width: 60%
 
 #. Select "Use Existing Drive".
 
-   .. figure:: /_static/images/setup/screen1-restore_or_useexisting.jpg
+   .. figure:: /_static/images/setup/screen3-use_existing.png
       :width: 60%
 
-#. Select your Embassy data drive
+#. Select your Embassy data drive.
 
-   .. figure:: /_static/images/setup/screen4-select_storage.jpg
+   .. figure:: /_static/images/setup/screen4-use_existing_drive_selection.png
       :width: 60%
 
-#. Enter and confirm a new password.  This time, save it somewhere safe!
```

Comment:
Matt just added this, and it's probably fine - the user will have already set up a password previously, so they will understand the concept of the UI elements.  Also, they're hopefully self-explanatory

Reply:
Yeah the implication that they forgot their password isn't necessarily true though...

Reply:
Changed it anyway